yarn.lock

```diff
@@ -2025,30 +2025,11 @@
     "@emotion/weak-memoize" "^0.3.0"
```

Comment:
This yarn.lock change needs to be removed from the PR (likely due to running a production build)